api/tests/tooling/domain-builder/factory/build-skill.js

```diff
@@ -1,13 +1,12 @@
-const faker = require('faker');
 const Skill = require('../../../../lib/domain/models/Skill');
 
 const buildSkill = function buildSkill({
-  id = `rec${faker.random.uuid()}`,
-  name = `${buildRandomTubeName()}${faker.random.number(8)}`,
-  pixValue = faker.random.number(4),
-  competenceId = `rec${faker.random.uuid()}`,
+  id = 'recSK123',
+  name = '@saucisse',
```

Comment:
```suggestion
  name = '@saucisse4',
```
Sinon c'est pas un bon nom d'acquis

Reply:
`@saucisse6` me semble plus a propos ^^

Reply:
you mean `@sau6`  ?

Reply:
will do !